### PR TITLE
Location description

### PR DESCRIPTION
**Pull request by miazima**

### CODE REVIEW

File: src/lib/models/Suggestion.js [hunk 1]
```diff
@@ -136,6 +150,12 @@ export const DaysOfWeekSchema = Joi.object({
   Sun: Joi.bool().required(),
 })
 
+export const LocationDescriptionSchema = Joi.object({
+  description: Joi.string(),
+  postalCode: Joi.number(),
+  oneMapAddress: Joi.object(),
```
Comment (xkjyeah) [inline src/lib/models/Suggestion.js]: Maybe just "oneMapData"? Because it's not just the address

File: src/lib/endpoints/suggestions.js [hunk 2]
```diff
@@ -197,6 +200,8 @@ export function register(server, options, next) {
             .min(0)
             .max(24 * 3600e3),
 
+          boardDesc: LocationDescriptionSchema,
```
Comment (xkjyeah) [inline src/lib/endpoints/suggestions.js]: I would prefer giving this field the full (albeit leceh) name, i.e. "boardDescription"

File: test/suggestedRoutes.js [hunk 3]
```diff
@@ -44,6 +44,16 @@ lab.experiment("Suggested routes manipulation", function () {
...
       alight: Joi.attempt({lat: 1.35, lng: 103.75}, Joi.latlng()),
+      boardDesc: { 
+        postalCode: 4560, 
+        description: "456, A Street, S 4560", 
+        oneMapAddress: { postal: 4560, blk_no: 456, road_name: "A Street" }, 
+      },
+      alightDesc: { 
+        postalCode: 7560, 
+        description: "756, B Street, S 7560", 
+        oneMapAddress: { postal: 7560, blk_no: 756, road_name: "B Street" },
```
Comment (LoneRifle) [inline test/suggestedRoutes.js]: nitpick - is the object schema from onemap really in _ case?

Comment (miazima) [reply test/suggestedRoutes.js]: It is in uppercase actually, will modify the test accordingly

**Pull request merged**